33762.0.txt

```diff
@@ -0,0 +1,97 @@
+@0
+LOOP:
+MOV UP, ACC
+
+#Check if greater than 0, if not pass 0 down
+ZERO:
+JGZ ONE
+MOV 0, DOWN
+JMP LOOP
+
+#If >0 then must be a 1. Pass one down
+ONE:
+MOV 1, DOWN
+
+#Check next value. If 0, goto zero. Keeps checking until 0 is found.
+MOV UP, ACC
+JEZ ZERO
+MOV 0, DOWN
+JRO -3
+
+@1
```

Comment:
I appreciate these blocks are similar to `@0`, but still worth commenting -- especially to highlight the differences